pkg/metrics/metrics.go

```diff
@@ -299,3 +304,66 @@ func (c *Config) createEndpoint(subsets []v1.EndpointSubset) error {
 	}
 	return nil
 }
+
+// ensureServiceMonitor creates a serviceMonitor object in the operator namespace if it does not exist.
+func (c *Config) ensureServiceMonitor() error {
```

Comment:
Why is this being moved from the static config file?
That should be called out in the commit message